hacks/isolateVersion/7-updateDocusaurusConfig.sh

```diff
@@ -0,0 +1,65 @@
+notify "Updating docusaurus.config.js..."
+
+# Remove unused versionedLinks dependency
+sed -i '' '/const versionedLinks/d' docusaurus.config.js
+
+# Update `url` to include `unsupported`
+sed -i '' -e "s/docs.camunda.io/unsupported.docs.camunda.io/" docusaurus.config.js
+
+# Update `baseUrl` to specify isolated version
+sed -i '' -e "s/baseUrl: \"\\/\"/baseUrl: \"\/$ARCHIVED_VERSION\/\"/" docusaurus.config.js
+
+# Remove optimize docs plugin.
+#   1. Find the block that starts with the plugin declaration, and the closing braces afterward, and delete it.
```

Comment:
I tried to build this as one single `sed` command, but expanding beyond the matched block was difficult, and this seemed like a simpler way to accomplish the task.